po/fr.po

```diff
@@ -3793,7 +3793,7 @@ msgstr "le fichier %s n'est pas inscriptible par l'utilisateur"
 
 #: trailer.c:965
 msgid "could not open temporary file"
-msgstr "impossible de créer un fichier temporaire"
+msgstr "impossible d'ouvrir un fichier temporaire"
```

Comment:
Hmm... pour moi, ce serait plutôt l'inverse, si on voulait mettre l'accent sur le fait que c'était l'ouverture qui posait problème et pas le fichier, j'aurais imaginé un "could not open a temporary file". Dans la mesure ou le fichier est temporaire, et donc pas un fichier spécifique, et que par conséquent, ce n'est jamais le fichier en question qui est important, mais l'opération de le créer et de l'ouvrir, je pense qu'on peut dire que la distinction ne mérite pas qu'on s'y attarde longuement ^^. Laissons-ça tel quel, ça me va bien.